tools/downloadSyncClient.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# stop on errors (nonzero exit codes), uninitialized vars
+set -eu
+
+SYNC_PATH=app/extensions/brave/content/scripts/sync.js
+SYNC_URL=https://raw.githubusercontent.com/brave/sync-bundles/master/bundle.js
+
+curl -o $SYNC_PATH $SYNC_URL
```

Comment:
I added bundle.js and testbundle.js from brave/sync-bundles as the first release of brave/sync.

Download latest prod (non pre-release) version:
```sh
curl -O $(curl -s https://api.github.com/repos/brave/sync/releases/latest | grep \/bundle.js | head -n 1 | cut -d '"' -f 4)
```